homeassistant/components/roborock/sensor.py

```diff
@@ -235,3 +243,31 @@ def native_value(self) -> StateType | datetime.datetime:
         return self.entity_description.value_fn(
             self.coordinator.roborock_device_info.props
         )
+
+
+class RoborockSensorRoomEntity(RoborockCoordinatedEntity, SensorEntity):
+    """Representation of a room entity - it uses the coordinator so it is separate from others."""
+
+    _attr_entity_category = EntityCategory.DIAGNOSTIC
+    _attr_translation_key = "current_room"
+    _attr_key = "current_room"
```

Comment:
what's this for?

Reply:
tbh - I don't typically create an entity without a description - I just assumed it needed the key specified, does it not?

Reply:
the `key` of an `EntityDescription` has no use other than for identification. Integration use it in their unique_id and stuff